CruCentralCoast/Community/CommunityGroups/CommunityGroupsTVC.swift

```diff
@@ -38,32 +46,36 @@ class CommunityGroupsTVC: UITableViewController {
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         // #warning Incomplete implementation, return the number of rows
-        return 10
+        return dataArray.count
     }
     
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         return 260
     }
     
-//    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-//        guard let vc = UIStoryboard(name: "CommunityGroupDetails", bundle: nil).instantiateViewController(withIdentifier: "CommunityGroupDetailsVC") as CommunityGroupDetailsVC else {
-//            assertionFailure("Probably used the wrong storyboard name or identifier here")
-//        }
-//        return
-//    }
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard let vc = UIStoryboard(name: "CommunityGroupDetails", bundle: nil).instantiateViewController(withIdentifier: "CommunityGroupDetails") as? CommunityGroupDetailsVC else {
+            assertionFailure("Probably used the wrong storyboard name or identifier here")
+            return
+        }
+         self.navigationController?.present(vc, animated: true, completion: nil)
+    }
 
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "comCell", for: indexPath) as! CommunityTableViewCell
-
-        // Configure the cell...
+        let cell = tableView.dequeueCell(CommunityTableViewCell.self, indexPath: indexPath)        
         
-        cell.bannerImage.image = #imageLiteral(resourceName: "placeholder.jpg")
-        cell.bigLabel.text = "big label"
-        cell.smallLabel1.text = "label1"
-        cell.smallLabel2.text = "label2"
+        //cell.bannerImage.image = UIImage(named: dataArray[indexPath.row].imageLink!)
         
+        // i think i need to do a compact map or something here to get all strings in leader array
+        cell.bigLabel.text = dataArray[indexPath.row].name
         
+        let array = [dataArray[indexPath.row].dayOfWeek, dataArray[indexPath.row].meetingTime]
+        let dayAndTime = array.compactMap { $0 }
```

Comment:
What does this line do?

Reply:
those two lines grab the day of the week and meeting time items and puts them in a single variable for every community group